### PR TITLE
cartmanagement

### PR DESCRIPTION
**Pull request by preethasri**
added cart management to ecomm
features:
      *add to cart
      *remove from cart
      *move to wishlist from cart
      *added quantity and price to products

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ function App() {
          <Route path="/" element={<SignUp />} />
          <Route path="/logout" element={<Logout />} />
          <Route path="/wishlist" element={<WishList />} />
+         <Route path="/cart" element ={<CartManagement />} />
```
Comment (Chakravarthi0) [inline src/App.js:14]: suggestion: you can create a separate component for routes

File: src/pages/cartmanagement/cartProducts.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import "./cartmanagement.css"
+import { useWishList } from "../../context/WishListContext";
+import { useAuth } from "../../context/AuthContext";
+import { useCart } from "../../context/CartContext";
```
Comment (Chakravarthi0) [inline src/pages/cartmanagement/cartProducts.js:4]: You can import all three contexts in a single line.

File: src/pages/cartmanagement/cartmanagement.css [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    --secondary-color: #F05524;
+    --primary-btn: #6495ED;
+}
+
+
+
+  
+  .cart{
+  
+    width:800px;
```
Comment (Chakravarthi0) [inline src/pages/cartmanagement/cartmanagement.css:17]: You are uisng px unit in few places, you can use rem unit instead.

File: src/pages/productpage/allproducts/AllProducts.js [hunk 4]
```diff
@@ -5,6 +5,8 @@ import { useWishList } from "../../../context/WishListContext"
 import { addToWishList } from "../../../services/wishListService"
 import { removeFromWishList } from "../../../services/wishListService"
```
Comment (Chakravarthi0) [inline src/pages/productpage/allproducts/AllProducts.js:6]: you can import these services in one line.

File: src/pages/productpage/allproducts/allproducts.css [hunk 5]
```diff
@@ -75,18 +75,27 @@
     all:unset;
 }
 .add-to-wishlist i{
-    color: #000000;
+    color: red;
 }
 .add-to-wishlist i:hover {
     color: #e31b23;
```
Comment (Chakravarthi0) [inline src/pages/productpage/allproducts/allproducts.css:81]: You can use a variable, if you are re-using this color somewhere else

**Review by Chakravarthi0 — APPROVED** [review]

File: src/pages/productpage/allproducts/allproducts.css (hunk 5 above)
Comment (preethasri) [reply src/pages/productpage/allproducts/allproducts.css:81]: will work on this saran👍

File: src/components/navbar/navbar.js [hunk 6]
```diff
@@ -2,12 +2,14 @@ import "../navbar/navbar.css"
 import {Link,useNavigate} from 'react-router-dom';
 import { useAuth } from "../../context/AuthContext";
 import { useWishList} from "../../context/WishListContext";
+import { useCart } from "../../context/CartContext";
```
Comment (vijaytembugade) [inline src/components/navbar/navbar.js:5]: Suggesstion : You can use barrel import and export for context.

File: src/context/CartContext.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import {createContext} from 'react'
+import {useContext,useState,useEffect} from 'react'
```
Comment (vijaytembugade) [inline src/context/CartContext.js:3]: Both import from react only. Use VS code intelli or ES6/7 extesion for auth importing.

**Review by vijaytembugade — APPROVED** [review]
Good work. Use pretier for formatting.

File: src/context/CartContext.js (hunk 7 above)
Comment (preethasri) [reply src/context/CartContext.js:3]: will work on this vijay.thank you

**Pull request merged**